app/_src/guides/federate.md

```diff
@@ -0,0 +1,139 @@
+and then federate it to a multi-zone deployment.
+This way you can:
+- see your mesh deployment in one centralized place
+- connect multiple zones and introduce cross zone connectivity
+- manage policies that are pushed to all zones
+
+## Prerequisites
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to setup a zone control plane with demo application
+
+## Start a global control plane
+
+### Start a new Kubernetes cluster
+
+Currently, it's not possible to deploy global and zone control plane in the same Kubernetes cluster, therefore we need a new Kubernetes cluster.
+
+{% tip %}
+You can skip this step if you already have a Kubernetes cluster running.
+It can be a cluster running locally or in a public cloud like AWS EKS, GCP GKE, etc.
+{% endtip %}
+
+```sh
+kind create cluster --name=mesh-global
+```
+
+By default, Kind does not support LoadBalancer therefore we need to configure it by following this [guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/).
+Note that all public cloud providers support load balancers out of the box.
+
+### Deploy a global control plane
+
+```sh
+helm install --create-namespace --namespace {{site.mesh_namespace}} \
+--set {{site.set_flag_values_prefix}}controlPlane.mode=global \
+--set {{site.set_flag_values_prefix}}controlPlane.defaults.skipMeshCreation=true \
+{{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+We skip default mesh creation as we will bring mesh from zone control plane in the next steps.
+
+### Sync endpoint
+
+Find the external IP and port of the `{{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync` service in the `{{site.mesh_namespace}}` namespace:
+
+```sh
+kubectl get services -n {{site.mesh_namespace}}
+NAMESPACE     NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                                                                  AGE
+{{site.mesh_namespace}}   {{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync     LoadBalancer   10.105.9.10     35.226.196.103   5685:30685/TCP                                                           89s
+{{site.mesh_namespace}}   {{site.mesh_cp_name}}     ClusterIP      10.105.12.133   <none>           5681/TCP,443/TCP,5676/TCP,5677/TCP,5678/TCP,5679/TCP,5682/TCP,5653/UDP   90s
+```
+
+In this example the value is `35.226.196.103:5685`. You pass this as the value of `<global-kds-address>` when you federate the zone control plane.
+If you see `<Pending>` you either need to wait until load balancer is provisioned or you need to make sure that your Kubernetes cluster support provisioning load balancers. 
+
+## Copy resources from zone to global control plane
+
+To federate zone control plane without any traffic interruption, we need to copy resources like secrets, meshes etc.
+We can do this by executing the following command:
+
+```sh
+kubectl --context=mesh-zone port-forward svc/demo-app -n kuma-demo 5000:5000
```

Comment:
port forward without background will freeze, also I think you meant to port forward the CP:

```suggestion
kubectl --context=mesh-zone port-forward svc/demo-app -n kuma-demo 5000:5000 &
kubectl --context=mesh-zone port-forward svc/kuma-control-plane -n kuma-system 5681:5681 &
```

Reply:
I meant to do this yes. I'll separate this into two commands. I'd rather avoid `&` for less savvy bash users